upstream-sha256: dba294363e7d5765c18702bcc0ce3c2a562568d18b6758b2f7508b2444f49ca4
actual-sha256: b00ddce35f04267d0d69ffe30da1c76c8b135b6b3e2466b62950105b2db41972
--- upstream.expected
+++ tolk.actual
@@ -17,5 +17,5 @@
   float val8 = (*(data1_100+(alu0+80)));
   float val9 = (*(data1_100+(alu0+90)));
   float val10 = (*(data2_10+alu0));
-  *(data0_10+alu0) = (val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
+  *(data0_10+alu0) = (0.0f+val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
 }
